core/domain/exp_domain_test.py

```diff
@@ -4202,7 +4202,7 @@ def test_verify_all_states_reachable(self) -> None:
             'Please fix the following issues before saving this exploration: '
             '1. The following states are not reachable from the initial state: '
             'End 2. It is impossible to complete the exploration from the '
-            'following states: Stuck State, Introduction'):
```

Comment:
?

Reply:
Not sure, but it is like something flake. Passes many times but fails inbetween. Not under my work. Just solve it.

Reply:
Solved by Sahil PR. Thanks @heyimShivam, 

Please resolve comment.